ocaml/idl/datamodel_pool.ml

```diff
@@ -75,6 +75,21 @@ open Datamodel_types
       ~allowed_roles:_R_POOL_OP
       ()
 
+  (* This is a map of uuid -> cert_blob *)
+  let pool_certs = Map (String, String)
+
+  let pre_join_exchange_certificates = call
+      ~name:"pre_join_exchange_certificates"
+      ~in_oss_since:None
+      ~in_product_since:rel_next
+      ~params:[String, "uuid", "The uuid of the joiner";
```

Comment:
uuid of the joining host (joiner) - to make explicit the kind of the uuid

Reply:
I've removed the new API call